### DISCUSSION TITLE
Ability to update existing QSO with third-party data

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#702] [Ideas / Feature Requests]
When importing ADIF files from WSJT-X, there is no name or other information stored. Could there be a way to open up the QSO once in Cloudlog and update with data from QRZ.com?

**Comment by dg9vh** [comment]
Idea could be a 'fetch qrz.com data' button filling up empty fields and fields with data could be marked with some background color and as tool tip could be shown the fetched value so the user could modify such values manually if wanted.

**Comment by magicbug** [comment]
Yes agree would be a nice feature, although this would need to be presented to a user so they can decide whether they want to add the information to the QSO

**Comment by jsb2092** [comment]
Is this not already part of the lookup on qrz or hamqth import feature (though it has to be setup in the config.ini).  In the feature I was adding, it does update the name on import, but only if it's not already set, same with grid square.  Having a button could be good too for those that were missed, or if the feature isn't turned on.
